core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -311,6 +315,18 @@ class Main {
             val process = injector.getInstance(SearchProcessMonitor::class.java)
             process.saveOverall()
         }
+
+        private fun writeDependencies(injector: Injector) {
```

Comment:
can you add a couple of sentences in a comment here explaining what those "dependencies" are?

Reply:
fixed